changelog/13048.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:change
```

Comment:
Could you please remove the changelog entry ? We don't really do changelog entries for docs-only changes.

Reply:
Hi @pmmukh, removed the changelog file as you requested.